### ISSUE TITLE
Letter "v" has the wrong font weight in Semibold

### ISSUE THREAD

**Issue by dawe-rgb** [cadsondemak/prompt#6]
Hi cadsondemak,

thanks for the great font »prompt«! We've used it in a webproject and and we noticed that the small letter »v« has the wrong font weight in semibold. Is there a way you could fix it soon?

Thanks!
Kind regards,
Daniel

![Bildschirmfoto 2020-08-31 um 09 41 13](https://user-images.githubusercontent.com/56391227/91695790-f6564400-eb6e-11ea-96bf-4219063240c4.png)

**Comment by madeinffm** [comment]
Hey there! Any news on this?

**Comment by kenmcd** [comment]
Are you hosting the .woff2 font files on your own server?
Or using Goggle Fonts links?
Trying to get GF to update their hosted fonts is a big process.
But simply fixing these two fonts to use locally is not that big a deal.

**Comment by madeinffm** [comment]
Hey @kenmcd ,
thanks for your answer. We're hosting them on our own servers, so if you have a new version, that would be super helpful.

Cheers
florian

**Comment by kenmcd** [comment]
@madeinffm & @dawe-rgb 
OK. Give me a few days.
I forked it and opened-up the sources Sat. and they are quite a mess.
For some reason the SemiBold was originally created from a separate master. (also explains the kerning issue)
And the SemiBold lowercase **v** is the Medium glyph with just wider side bearings. Quite odd.
So the plan is to put the three main masters back together, fix the other issues, and then regenerate all the fonts. 
That should also create the SemiBold correctly.

This project appears to be abandoned so I will probably post the new fonts in my fork, or ...
And I will change the name to _Prompt 1,_ still version one features but fixed.
Or maybe I'll add more features and make it _Prompt 2_ ... dunno.
Makes me crazy when software updates re-install the old un-fixed fonts, so now I always rename them.
And it makes it clear to others that something has changed.

Google Fonts makes at least four different versions of the `.woff2` files with different character sets.
To minimize the file size served.
Do you know which ones you are using?
Does the size matter that much to you?
Not really an issue, I will just create `.woff2` versions with the full font (which is usual here).

**Comment by madeinffm** [comment]
Hey @kenmcd,
thank you very much for your detailed answer!!! Super cool! 
Size does not matter that much really, I don't expect the full package to be that anyhow.

Looking forward to the files,
cheers
florian

**Comment by kenmcd** [comment]
The short version ...
SemiBold Regular and Italic fonts with a fixed lowercase v are attached below.
Includes `.woff2` and `.ttf` files.
Do not know why the `Prompt-SemiBoldItalic.ttf` file is so big. AIDK.
[Prompt-SemiBold-fonts-fixed.zip](https://github.com/cadsondemak/prompt/files/5638178/Prompt-SemiBold-fonts-fixed.zip)

The kerning is still wacko, but the v weight is right.
![Prompt-SemiBold-Original-vs-Fixed-lowercase-v](https://user-images.githubusercontent.com/844920/101071830-fee55e80-3551-11eb-8b15-6cebab0e942f.png)


The long version ...
Wadda mess.
Merged the "source" masters, created the instances for each weight, exported the fonts,
and got a bunch of errors regarding mis-matches in the characters present in each master.
Mostly in the Vietnamese, but also some in the Latin characters.
The resulting fonts were missing the lowercase a and m for example.
So I thought, these are worse than I expected, and need major fixing.
But, _the SemiBold lowercase v was there,_ and it is the right weight.
So I decided to just plug that one character into the original fonts.
That was relatively painless.
So give these a try and let me know how it goes.

Prompt is often mentioned when people are looking for a FOSS alternative to Avenir.
I came here looking for an open source font I can modify to create my own open source extended Avenir replacement.
This looks like it may not be a good candidate. 
Dunno.
I'm off. Got other stuff to do ...

**Comment by madeinffm** [comment]
Hey @kenmcd ,
sorry for the late answer but WOW, thank you very much for your work!!! We'll give it a try! Thank you so much for your efforts! Much appreciated!

Cheers!
Florian

**Comment by patricekaufmann** [comment]
Hello, thanks for the updated version of the font. The v for sure looks better now :)

However there is a difference compared to the font provided by Google. It's a checkbox that sets a bit for "use typo metrics". The fonts provided by google have this setting unchecked while the improved TTF has it checked which leads to different rendering results in browsers, e.g. small offsets on the glyphs. I used the FontForge editor to manually uncheck this checkbox. Afterwards the font is appearing the same as the one provided by google. 

I can provide an image which shows the checkbox, unfortunately it's in german, left old, right new:
![metric](https://user-images.githubusercontent.com/41791606/105048707-247be800-5a6c-11eb-837b-dcc880abb222.png)

Do you happen to know if this is something Google does to all fonts when providing them?

As for file size of the woff and woff2 formats in this ticket: That's most likely a charset thing and the larger woff files contain more glyphs.

Regards,
Patrice

**Comment by madeinffm** [comment]
Hey @patricekaufmann ,
thanks for your insights!
Cheers,
Florian

**Comment by kenmcd** [comment]
> Do you happen to know if this is something Google does to all fonts when providing them?

Yes, GF does make changes to fix fonts - with no notice or change in version number.
They call them _hotfixes_, I call them _stealth updates._.
So they fixed the vertical metrics to meet current guidelines.
And I did not notice.
That is part of the more current GF standards.
Standards which these fonts would never pass today.

In hindsight I should have just started with the fonts, rather than the "sources."
Would have been easier, and quicker, and less error prone.

**Comment by raabmar** [comment]
Found the same problem in 2024, started researching how to report broken Google Fonts and came here. Now I am seeing this was already reported many years ago but the official font is sill broken which can be seen here also https://fonts.google.com/
